pkg/acsengine/k8s_versions.go

```diff
@@ -243,25 +244,26 @@ func getK8sVersionComponents(version string, overrides map[string]string) map[st
-			DefaultTillerAddonName:       k8sComponentVersions["1.11"]["tiller"],
-			DefaultReschedulerAddonName:  k8sComponentVersions["1.11"]["rescheduler"],
-			DefaultACIConnectorAddonName: k8sComponentVersions["1.11"]["aci-connector"],
-			"nodestatusfreq":             k8sComponentVersions["1.11"]["nodestatusfreq"],
-			"nodegraceperiod":            k8sComponentVersions["1.11"]["nodegraceperiod"],
-			"podeviction":                k8sComponentVersions["1.11"]["podeviction"],
-			"routeperiod":                k8sComponentVersions["1.11"]["routeperiod"],
-			"backoffretries":             k8sComponentVersions["1.11"]["backoffretries"],
-			"backoffjitter":              k8sComponentVersions["1.11"]["backoffjitter"],
-			"backoffduration":            k8sComponentVersions["1.11"]["backoffduration"],
-			"backoffexponent":            k8sComponentVersions["1.11"]["backoffexponent"],
-			"ratelimitqps":               k8sComponentVersions["1.11"]["ratelimitqps"],
-			"ratelimitbucket":            k8sComponentVersions["1.11"]["ratelimitbucket"],
-			"gchighthreshold":            k8sComponentVersions["1.11"]["gchighthreshold"],
-			"gclowthreshold":             k8sComponentVersions["1.11"]["gclowthreshold"],
+			"dns":                              k8sComponentVersions["1.11"]["kube-dns"],
+			"addonmanager":                     k8sComponentVersions["1.11"]["addon-manager"],
+			"dnsmasq":                          k8sComponentVersions["1.11"]["dnsmasq"],
+			"pause":                            k8sComponentVersions["1.11"]["pause"],
+			DefaultTillerAddonName:             k8sComponentVersions["1.11"]["tiller"],
+			DefaultReschedulerAddonName:        k8sComponentVersions["1.11"]["rescheduler"],
+			DefaultACIConnectorAddonName:       k8sComponentVersions["1.11"]["aci-connector"],
+			"nodestatusfreq":                   k8sComponentVersions["1.11"]["nodestatusfreq"],
+			"nodegraceperiod":                  k8sComponentVersions["1.11"]["nodegraceperiod"],
+			"podeviction":                      k8sComponentVersions["1.11"]["podeviction"],
+			"routeperiod":                      k8sComponentVersions["1.11"]["routeperiod"],
+			"backoffretries":                   k8sComponentVersions["1.11"]["backoffretries"],
+			"backoffjitter":                    k8sComponentVersions["1.11"]["backoffjitter"],
+			"backoffduration":                  k8sComponentVersions["1.11"]["backoffduration"],
+			"backoffexponent":                  k8sComponentVersions["1.11"]["backoffexponent"],
+			"ratelimitqps":                     k8sComponentVersions["1.11"]["ratelimitqps"],
+			"ratelimitbucket":                  k8sComponentVersions["1.11"]["ratelimitbucket"],
+			"gchighthreshold":                  k8sComponentVersions["1.11"]["gchighthreshold"],
+			"gclowthreshold":                   k8sComponentVersions["1.11"]["gclowthreshold"],
+			DefaultNVIDIADevicePluginAddonName: k8sComponentVersions["1.11"]["nvidia-device-plugin"],
```

Comment:
just curious, does NDP v1.10 work in k8s 1.11? when i tested NDP v1.9 with k8s 1.10 a while back, it didn't work

Reply:
It appears to work the same way it does in 1.10